### PR TITLE
[R4R] add pyUSID

### PR DESCRIPTION
**Pull request by carlodri**
https://github.com/pycroscopy/pyUSID

### CODE REVIEW

**Comment by CompPhysChris** [comment]
I'm good with being a maintainer.

**Comment by ssomnath** [comment]
I am OK with being a maintainer as well

**Comment by carlodri** [comment]
@conda-forge/staged-recipes this can be reviewed and, if ok, merged.

File: recipes/pyusid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
```
[the remark this answers is not in the document]
Comment (carlodri) [reply recipes/pyusid/meta.yaml]: looks like we need it to build: https://github.com/pycroscopy/pyUSID/blob/master/setup.py#L62

File: recipes/pyusid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - numpy >=1.13.0
+    - h5py >=2.6.0
+    - matplotlib >=2.0.0
+    - psutil
+    - six
+    - joblib >=0.11.0
+    - ipywidgets >=5.2.2
+    - ipython >=5.1.0,<6  # [py<34]  # IPython 6.0+ does not support Python 2.6, 2.7, 3.0, 3.1, or 3.2
+    - ipython >=6.0  # [py>=34]  # Beginning with IPython 6.0, Python 3.3 and above is required.
+    - pyqt >=5  # [py35]
```
[the remark this answers is not in the document]
Comment (carlodri) [reply recipes/pyusid/meta.yaml]: @ssomnath @CompPhysChris can you comment on this?

[the remark this answers is not in the document]
Comment (carlodri) [reply recipes/pyusid/meta.yaml]: not sure as to why it is like that, but now I  changed it to reflect the `setup.py`: https://github.com/pycroscopy/pyUSID/blob/master/setup.py#L29

File: recipes/pyusid/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/pyusid/meta.yaml]: huumm.. that's true. Thanks for checking it! :)

**Comment by scopatz** [comment]
Does this look good to you @marcelotrevisani?

**Comment by carlodri** [comment]
LGTM, if @marcelotrevisani approves you can merge, thanks.

**Pull request merged**